webpack.config.base.js

```diff
@@ -1,18 +1,35 @@
 'use strict';
 
+var _ = require('lodash');
+var path = require('path');
 var webpack = require('webpack');
 
 module.exports = {
   module: {
-    loaders: [
-      { test: /\.js$/, loaders: ['babel-loader'], exclude: /node_modules/ }
-    ]
+    loaders: [{
+      test: /\.js$/,
+      loaders: ['babel-loader'],
+      include: [
+        path.resolve(__dirname, 'src'),
+        path.resolve(__dirname, 'node_modules', 'lodash')
```

Comment:
> :question: Is this necessary because of NormalModuleReplacementPlugin below?

I believe so. We can try removing lodash's path from it to see if it still works.